app/src/main/java/org/fossasia/openevent/fragments/LocationsFragment.java

```diff
@@ -47,12 +48,15 @@
 
     private SearchView searchView;
 
+    private TextView noMicrolocationsView;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'noMicrolocationsView' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406630555/issues/source?bid=3233850&fileBranchId=3233897#l51)